guides/manual-installation.md

```diff
@@ -5,255 +5,223 @@
+      ```sql
+      \c wanda
+      GRANT ALL ON SCHEMA public TO wanda_user;
+      \c trento
+      GRANT ALL ON SCHEMA public TO trento_user;
+      \c trento_event_store;
+      GRANT ALL ON SCHEMA public TO trento_user;
+      \q
+      ```
+    You can exit from the `psql` console and `postgres` user.
+
+1.  Allow the PostgreSQL database to receive connections for the respective databases and users adding the following in `/var/lib/pgsql/data/pg_hba.conf`:
+
+    ```bash
+    host   wanda                      wanda_user    0.0.0.0/0   md5
+    host   trento,trento_event_store  trento_user   0.0.0.0/0   md5
+    ```
+
+    > **Note:** The `pg_hba.conf` file works in a sequential fashion. This means, that the rules positioned on the top have preference over the ones coming next. This examples shows a pretty permissive address range, so in order to work, they entries must be written in the top of the `host` entries. Find additional information in the [pg_hba.conf](https://www.postgresql.org/docs/current/auth-pg-hba-conf.html) documentation.
+
+
+1.  Allow PostgreSQL to bind on all network
+    interfaces in `/var/lib/pgsql/data/postgresql.conf` by changing the following line:
+
+    ```bash
+    listen_addresses = '*'
+    ```
+
+1.  Restart PostgreSQL to apply the changes:
+
+    ```bash
+    systemctl restart postgresql
+    ```
 
 ### Install RabbitMQ
 
-```bash
-zypper install rabbitmq-server
-```
+1.  Install RabbitMQ server:
+    ```bash
+    zypper install rabbitmq-server
+    ```
 
-As the agent needs to reach RabbitMQ, allow connections from external hosts.
-Modify `/etc/rabbitmq/rabbitmq.conf` and ensure the following lines are present:
+1.  The agent needs to reach RabbitMQ, allow connections from external hosts by modifying `/etc/rabbitmq/rabbitmq.conf`:
+    ```bash
+    listeners.tcp.default = 5672
+    ```
 
-```bash
-listeners.tcp.default = 5672
-```
+1.  Add an exception on firewalld **(ONLY if firewalld is running)**:
 
-Add an exception on firewalld **(ONLY if firewalld is running)**:
+    ```bash
+    firewall-cmd --zone=public --add-port=5672/tcp --permanent;
+    firewall-cmd --reload
+    ```
 
-```bash
-firewall-cmd --zone=public --add-port=5672/tcp --permanent;
-firewall-cmd --reload
-```
+1.  Enable RabbitMQ service:
 
-As the agent needs to reach RabbitMQ, allow connections from external hosts:
+    ```bash
+    systemctl enable --now rabbitmq-server
+    ```
 
-```bash
-systemctl enable --now rabbitmq-server
-```
+#### Configure RabbitMQ
 
-### Configure RabbitMQ
+In order to configure RabbitMQ for a production system, follow the official suggestions [RabbitMQ guide](https://www.rabbitmq.com/production-checklist.html)
 
-> Note: In order to configurate RabbitMQ for a production system, follow the official suggestions [RabbitMQ guide](https://www.rabbitmq.com/production-checklist.html)
+1.  Create a new RabbitMQ user:
+    ```bash
+    rabbitmqctl add_user trento_user trento_user_password
+    ```
```

Comment:
Can this be executed as normal user or do you need root privileges?

Reply:
It depends if this is a virtual machine, a cloud machine, normally we run everything with root access level.

Reply:
Okay. Would sudo be useful then?